packages/orbit-components/src/Stepper/README.md

```diff
@@ -79,12 +82,11 @@ Table below contains all types of the props available in `StepperStateless` comp
 | ariaLabelledby       | `string`                    |         | Optional prop for `aria-labelledby` value. See accessibility tab.                         |
 | descriptionDecrement | `string`                    |         | Optional prop for `aria-describedby` value for decrement `Button`. See accessibility tab. |
 | descriptionIncrement | `string`                    |         | Optional prop for `aria-describedby` value for increment `Button`. See accessibility tab. |
-| ariaDescribedby      | `string`                    |         | Optional prop for `aria-describedby` value for input field. See accessibility tab.        |
```

Comment:
What happened here? 🤔

Reply:
this dissappeared when solving conflicts. should be fixed.